libs/ui/elements/src/lib/ui-elements.module.ts

```diff
@@ -26,6 +27,7 @@ import { UiSearchModule } from '@geonetwork-ui/ui/search'
     UtilSharedModule,
     UiSearchModule,
     RouterModule,
+    FeatureCatalogModule,
```

Comment:
no `ui` should not depend on `feature`
put the model elsewhere if it's what you need.

Reply:
Yes, I see, removed this